torch/autograd/gradcheck.py

```diff
@@ -616,12 +675,13 @@ def vjp_fn(grad_output):
                                    retain_graph=True, allow_unused=True)
     # Compute everything twice to check for nondeterminism (which we call reentrancy)
     if fast_mode:
-        vjps1 = _get_analytical_vjps_wrt_specific_output(vjp_fn, output.clone(), v)
-        vjps2 = _get_analytical_vjps_wrt_specific_output(vjp_fn, output.clone(), v)
+        vjps1 = _get_analytical_vjps_wrt_specific_output(vjp_fn, output, v)
+        vjps2 = _get_analytical_vjps_wrt_specific_output(vjp_fn, output, v)
```

Comment:
There is no reason to clone when _get_analytical_vjps_wrt_specific_output only uses `output.shape`.